tools/analyzer_plugin/test/integration/diagnostics/exhaustive_deps_test_cases.dart

```diff
@@ -654,8 +654,8 @@ final Map<String, List<Map<String, Object>>> tests = {
           final maybeRef1 = useSomeOtherRefyThing();
           var state1 = useState<dynamic>(null);
           var state2 = over_react.useState<dynamic>(null);
-          var state3 = useReducer(null, null);
-          var state4 = over_react.useReducer(null, null);
+          var state3 = useReducer((_, __) => null, null);
```

Comment:
Is this related to the null safe stuff? Or just a typo from the original implementation?